app/views/shared/forms/_identify_types_of_outgoings_form.html.erb

```diff
@@ -28,20 +32,22 @@
     </fieldset>
   <% end %>
 
-  <details class="govuk-details govuk-!-margin-top-4" data-module="govuk-details">
-    <summary class="govuk-details__summary">
-          <span class="govuk-details__summary-text">
-            <%= t('.expanded_explanation.heading') %>
-          </span>
-    </summary>
+  <% if citizen %>
```

Comment:
Could you just do `<% if journey == :citizen %>` here, and then the remove the code on lines 1-4? You're not using the `citizen` variable anywhere else so it seems a bit unncessary.

Reply:
True, I just wanted to make it more visible by placing the variable at the top. Since I have tests in, It'll probably fail if the journey variable wasn't defined or deleted somehow.

I'll change it, thanks 👍